src/index.js

```diff
@@ -195,6 +198,12 @@ module.exports = {
       revision: pkg.revision || '1'
     }
   },
+  getDepends: dependencies.getDepends,
+  getGConfDepends: dependencies.getGConfDepends,
+  getGTKDepends: dependencies.getGTKDepends,
+  getTrashDepends: dependencies.getTrashDepends,
+  getUUIDDepends: dependencies.getUUIDDepends,
+  mergeUserSpecified: dependencies.mergeUserSpecified,
```

Comment:
I don't agree with exporting all these internal functions, we should only export the ones needed by the other modules. The modules only need `getTrashDepends` and `getDepends`

Reply:
They're all used in installer modules:

https://github.com/electron-userland/electron-installer-snap/blob/4ce638e1f9243f9eb0a0763b46fc9fcb36cd742e/src/yaml.js#L165-L171

Reply:
Fun fact: `electron-installer-snap` is the first published module to use `electron-installer-common`. :smile:

Reply:
Oh! I totally forgot about `-snap`. I was only thinking of `-debian` and `-redhat`. Thanks for pointing that out.